README.md

```diff
@@ -18,24 +18,25 @@ This repo contains a [template `local_nb_nodes.json`](/local_nb_nodes.json) file
 
 Examples:  
 
-`local_nb_nodes.json` with one local node
+`local_nb_nodes.json` with one local node API running on `http://localhost:8000`
 ```json
 {
-    "NodeName": "First Node",
-    "ApiURL": "https://firstnode.org"
+    "NodeName": "Local graph",
+    "ApiURL": "http://host.docker.internal:8000"
 }
 ```
+_**NOTE:** If the local node API(s) you are federating over is running on the same host machine as the federation API (e.g., the URL to access the node API is `http://localhost:XXXX`), in `local_nb_nodes.json` you **must** replace `localhost` with `host.docker.internal` in the corresponding `"ApiURL"`, as shown above._
```

Comment:
```suggestion
_**NOTE:** If the local node API(s) you are federating over is running on the same host machine as the federation API 
(e.g., the URL to access the node API is `http://localhost:XXXX`), 
in `local_nb_nodes.json` you **must** replace `localhost` 
with `host.docker.internal` in the corresponding `"ApiURL"`, 
as shown above 
(see also the [Docker docs](https://docs.docker.com/engine/reference/commandline/run/#add-host))._
```

Reply:
:cherries: 
- docs link for context
- shorter line -> nicer diff in the future